code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -0,0 +1,71 @@
+/datum/job/mining
```

Comment:
i'm going to trust this file is working and has no unintended changes without reading every line